bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java

```diff
@@ -1318,4 +1322,11 @@ private BookieNode replaceToAdherePlacementPolicyInternal(
         }
         throw new BKNotEnoughBookiesException();
     }
+
+    protected String getLocalRack(BookieNode node) {
+        if (null == node || null == node.getAddr()) {
+            return UNKNOWN_RACK;
+        }
+        return node.getNetworkLocation();
```

Comment:
If the network location is `/default-rack`, shall we think of it as UNKNOWN_RACK?